TUTORIAL.md

```diff
@@ -0,0 +1,396 @@
+# CesiumJS and Webpack
+
+[Webpack](https://webpack.js.org/) is a popular and powerful tool for bundling JavaScript modules. It allows developers to structure their code and assets in an intuitive way and to load different kinds of files as needed with simple require statements. When building, it will trace code dependencies and pack these modules into one or more bundles that are loaded by the web browser.
+
+In this tutorial, we will build a simple web app from the ground up using webpack, and then cover the steps to integrate the [Cesium npm module](https://www.npmjs.com/package/cesium). This is a good place to start if you’d like to use CesiumJS to develop a web application. If you’re new to Cesium and are looking to learn to build your first sample app, take a look at our [Getting Started Tutorial](https://cesium.com/learn/cesiumjs-learn/cesiumjs-quickstart/).
+
+## Prerequisites
+- A basic understanding of the command line, JavaScript, and web development.
+- An IDE or code editor. Developers on the Cesium team members use Visual Studio Code, but a minimal code editor such as Sublime Text will also work.
+- Node.js installed. We recommend using the latest LTS version.
+
+## Initialize an app with npm
+
+1. Create a new directory for your app.
+2. Open a console window and navigate to the directory.
+3. Run `npm init` and enter in the requested details about your application. If you are unsure about any prompts, press `Enter` to use the default value. You can modify these details at any point in the `package.json` file that `npm init` creates.
+
+## Create the app code
+
+1. Create a `src` directory for your app code. When we build the app, webpack will automatically produce distribution files in a directory named `dist`.
+2. Create the files `index.html` and `index.js` in the `src` directory added in step 1.
+3. Add the following code to `src/index.html`
+```
```

Comment:
In Markdown, when adding a code block within a list, add the 4 spaces as a prefix. Makes for better formatting. Also give it a hint for content, in this case html.

Reply:
I'm pretty sure the VS Code extension handles it correctly, but try it in GitHub web editor + preview as well.

You can give markdown hints about syntax formatting. https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks